tests/integration/goldens/credentials/google/iam/credentials_v1/services/iam_credentials/transports/grpc.py

```diff
@@ -186,6 +246,11 @@ def __init__(self, *,
                 ],
             )
 
+        self._grpc_channel = self._base_channel
+        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(std_logging.DEBUG):  # pragma: NO COVER
```

Comment:
I suggest checking here whether any logging levels are handled at all, so using either `std_logging.NOTSET` or the next highest priority, the numeric `1` ([ref](https://docs.python.org/3/library/logging.html#levels))

Reply:
addressed.